src/pages/workspace/companyCards/assignCard/AssignCardFeedPage.tsx

```diff
@@ -21,14 +21,20 @@ import TransactionStartDateStep from './TransactionStartDateStep';
 type AssignCardFeedPageProps = PlatformStackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.COMPANY_CARDS_ASSIGN_CARD> & WithPolicyAndFullscreenLoadingProps;
 
 function AssignCardFeedPage({route, policy}: AssignCardFeedPageProps) {
-    const [assignCard] = useOnyx(ONYXKEYS.ASSIGN_CARD);
+    const [assignCard] = useOnyx(ONYXKEYS.ASSIGN_CARD, {canBeMissing: true});
     const currentStep = assignCard?.currentStep;
 
     const feed = decodeURIComponent(route.params?.feed) as CompanyCardFeed;
     const backTo = route.params?.backTo;
     const policyID = policy?.id;
-    const [isActingAsDelegate] = useOnyx(ONYXKEYS.ACCOUNT, {selector: (account) => !!account?.delegatedAccess?.delegate});
-
+    const [isActingAsDelegate] = useOnyx(ONYXKEYS.ACCOUNT, {selector: (account) => !!account?.delegatedAccess?.delegate, canBeMissing: true});
+    const firstAssigneeEmail = useRef(assignCard?.data?.email);
+    /* eslint-disable react-compiler/react-compiler */
+    if (!firstAssigneeEmail.current) {
+        firstAssigneeEmail.current = assignCard?.data?.email;
+    }
```

Comment:
Let's create a custom hook for this: `useInitial` with description "A hook that returns the initial non-undefined value of a variable"